### ISSUE TITLE
WebGL Errors

### ISSUE THREAD

**Issue by arkham81a** [CBenni/mt2#30]
Ever since the latest build I've been running into almost no noticeable issues but I keep getting the "WebGL hit a snag" Message. Photo of the error message, console errors, and paste-bin config is below.

Error Prompt - https://i.imgur.com/URSNxCj.png

Errors - https://i.imgur.com/9V17LWy.png
 - https://i.imgur.com/Di35WJ3.png

Config - https://pastebin.com/mfrqrBRb

**Comment by CBenni** [comment]
I feel like one of your extensions is the culprit, which do you have installed?

**Comment by CBenni** [comment]
https://i.imgur.com/Di35WJ3.png that error is fixed, had nothing to do with the webgl errors however.

In what circumstance does the webgl error occur?

**Comment by arkham81a** [comment]
I have FFZ, FFZ:AP, and a Chrome extension called savvy time that's a simple time zone converter (link here https://bit.ly/2vxRb9F) - , I do have an Adblocker but it has the MT2 Site Whitelisted. This error most commonly occurs when I minimize and maximize the window with the whispers tab open. However it hasn't happened in several hours since I reinstalled Savvytime, because looking at the errors it did look like an extension was causing it and I reinstalled the most recent one I installed, so it probably was that.
